zcash_test_vectors/transaction_v6.py

```diff
@@ -81,6 +81,38 @@ def __bytes__(self):
         return ret
 
 
+class ActionGroupDescription(object):
+    def __init__(self, rand, anchor_orchard, proofs_orchard, is_coinbase):
+        self.vActionsOrchard = []
+        # There must always be a non-zero number of Action Descriptions in an Action Group.
+        for _ in range(rand.u8() % 4 + 1):
+            self.vActionsOrchard.append(OrchardZSAActionDescription(rand))
+        self.flagsOrchard = rand.u8()
```

Comment:
you don't need to asign `self.flagsOrchard` to a temp value. Just use `rand.u8()` when needed

Reply:
Yes, true. Fixed.